REWRITES.md

```diff
@@ -0,0 +1,14 @@
+> This document outlines the changes between the old [Holoitems](https://github.com/dlee13/OldHoloItems) and the rewrite.
+> Sorted Chronologically
+
+# Magnet
+(Please fill in, I haven't been following at this point)
+
+# Tide Rider
+Fixed a bug where players could scale a wall with the Tide Rider.
```

Comment:
The old item reduces by 1 durability each time you stop using it. The new item reduces by 1 durability when you start using it and reduces by another 1 durability when you stop. The old item was very weird about adding and removing enchantments on the itemstack.